### ISSUE TITLE
Don't report "should be a function, not a wire" error when it's because the submodule didn't resolve

### ISSUE THREAD

**Issue by VonTum** [pc2/sus-compiler#115]
```sus
module TEST {
    NON_EXISTING_MODULE m    // <- Already have an error that sais this couldn't be resolved

    input float f
    when m.may_push {
        m.push(fp_mul(f, f))    // <- on this line, we get a second error "A function call expects this to be an interface, but found a regular wire"
    }
}
```

**Comment by VonTum** [comment]
Hmm, after closer inspection it seems infeasible to fix this, because it interprets m as a struct, so m.push is a valid syntax to access the NON_EXISTING_MODULE struct's "push" field.
